### PR TITLE
Add mtomco to the team page

### PR DESCRIPTION
**Pull request by michaltomco**

### CODE REVIEW

File: _config/users.yml [hunk 1]
```diff
@@ -66,6 +66,13 @@
   twitterUsername: dupliaka
   employedBy: Red Hat
 
+  userId: michaltomco
```
Comment (ge0ffrey) [inline _config/users.yml:69]: `- ` missing in front of userId. That breaks yml, causing the website to fail.
Once fixed, this is good to merge.

Comment (ge0ffrey) [reply _config/users.yml:69]: that dash indicates that this is the start of a new element in the users array

Comment (rsynek) [reply _config/users.yml:69]: I was just going to build it locally. Thanks!

Comment (michaltomco) [reply _config/users.yml]: Thank you for noticing Geoffrey!

**Pull request merged**